dockerfiles/platform-build-centos-7/Dockerfile

```diff
@@ -62,15 +52,33 @@ ENV LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
 ENV PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
 ENV ACLOCAL_PATH=/usr/local/share/aclocal:$ACLOCAL_PATH
 
-RUN curl -L https://npmjs.org/install.sh | bash
+RUN echo "source /opt/rh/devtoolset-9/enable" >> /etc/bashrc
+SHELL ["/bin/bash", "--login", "-c"]
 
-ENV VCPKG_BINARY_SOURCES="clear;nuget,GitHub,read"
-ENV VCPKG_NUGET_REPOSITORY=https://github.com/hpcc-systems/vcpkg
+RUN . <(curl https://aka.ms/vcpkg-init.sh -L)
+RUN . ~/.vcpkg/vcpkg-init fetch cmake
+RUN ln -s `. ~/.vcpkg/vcpkg-init fetch cmake | tail -n 1` /usr/local/bin/
+RUN ln -s `. ~/.vcpkg/vcpkg-init fetch node | tail -n 1` /usr/local/bin/
+RUN ln -s `. ~/.vcpkg/vcpkg-init fetch node | tail -n 1 | sed -e 's!bin/node!bin/npm!g'` /usr/local/bin/
```

Comment:
question: guaranteed to succeed I presume because base image is known not to have these already in /usr/local/bin/,
but perhaps there should be a proceeding rm -r /usr/local/bin/cmake etc. ?

Reply:
Yes `/usr/local/bin` is known to be empty at this point - I don't think adding a `rm` is useful during a docker build image.